extractor/BUILD.bazel

```diff
@@ -27,9 +28,7 @@ TEST_DARS = [
     daml_compile(
         name = darmod,
         srcs = ["src/test/resources/damls/%s.daml" % darmod],
-        # FixMe: https://github.com/digital-asset/daml/issues/2289
-        # change version to lf_latest_version when freezing numeric in the next language version
-        target = lf_dev_version,
+        target = lf_latest_version,
```

Comment:
`Numeric` is already in DAML-LF 1.8, isn't it? So, I guess we just missed this.

Reply:
Yes.